### DISCUSSION TITLE
abnormal loss from train_sft.py

### DISCUSSION THREAD

**Discussion by houghtonweihu** [hpcaitech/ColossalAI#3635] [Community | Q&A]
I tried train_sft.py with model = gpt2, but got the following warning:

fit colossalai - colossalai - WARNING: batch_id:2, abnormal loss: 10.97655200958252

What is the meaning of abnormal loss?

   Thanks!

**Comment by binmakeswell** [comment]
Hi @houghtonweihu You can refer to [the example loss](https://github.com/hpcaitech/ColossalAI/tree/main/applications/Chat/examples#experiment-result).  Obviously, your loss is too high.

**Reply by houghtonweihu** [reply]
Thanks @binmakeswell for your reply! Did you try the train_sft.py with model = gpt2? What I try to learn is: does train_sft.py also work for gpt2.
